mix.exs

```diff
@@ -66,11 +66,12 @@ defmodule Supavisor.MixProject do
       # pooller
       # {:poolboy, "~> 1.5.2"},
       {:poolboy, git: "https://github.com/abc3/poolboy.git", tag: "v0.0.2"},
-      {:partisan, git: "https://github.com/lasp-lang/partisan.git", tag: "v5.0.0-rc.12"},
+      {:partisan, git: "https://github.com/lasp-lang/partisan.git", tag: "v5.0.0-rc.13"},
```

Comment:
let's not touch partisan yet, because it will be removed anyway

Reply:
I done it to support OTP 26 (though IIRC it still do not resolve an issue). I can roll it back.